words/objects.txt

```diff
@@ -2087,6 +2662,7 @@ sunstone
 supermarket
 supernova
 supply
+support
```

Comment:
I guess I missed this one first time around. Might be better to stay away from department names?